python-avd/pyavd/_eos_designs/structured_config/underlay/ethernet_interfaces.py

```diff
@@ -158,23 +158,26 @@ def ethernet_interfaces(self: AvdStructuredConfigUnderlay) -> list | None:
                     # Render port-channel member
                     ethernet_interface.update(
                         {
-                            "type": "port-channel-member",
                             "channel_group": {
                                 "id": int(channel_group_id),
                                 "mode": "active",
                             },
                         },
                     )
                     if get(link, "inband_ztp_vlan"):
-                        ethernet_interface.update({"mode": "access", "vlans": link["inband_ztp_vlan"]})
+                        ethernet_interface.update({"switchport": {"mode": "access", "access_vlan": link["inband_ztp_vlan"]}})
```

Comment:
Same as the other one, we need `enabled: True` here to fix the inband ztp issue.

Reply:
Fixed